content/docs/get-started/aws/deploy-changes.md

```diff
@@ -12,57 +12,369 @@ menu:
- +   ├─ aws:kms:Key       my-key          created
- ~   └─ aws:s3:Bucket     my-bucket       updated     [diff: +serverSideEncryptionConfiguration]
+     Type                     Name            Status      Info
+     pulumi:pulumi:Stack      quickstart-dev
+  +   └─ aws:s3:BucketObject  index.html                  created
 
 Outputs:
-    bucket_name: "my-bucket-de014a8"
+    bucketName: "my-bucket-68e33ec"
 
 Resources:
     + 1 created
-    ~ 1 updated
-    2 changes. 1 unchanged
+    2 unchanged
+
+Duration: 6s
+```
+
+Once the update has completed, rerun the command to list the contents of your bucket.
+
+{{< chooser language "javascript,typescript,python,go,csharp" / >}}
+
+{{% choosable language javascript %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucketName)
+```
+
+{{% /choosable %}}
+
+{{% choosable language typescript %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucketName)
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucket_name)
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output bucketName)
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```bash
+$ aws s3 ls $(pulumi stack output BucketName)
+```
+
+{{% /choosable %}}
+
+Notice that your index.html file has been added:
+
+```bash
+2020-08-27 12:30:24         68 index.html
+```
+
+{{% choosable language javascript %}}
+Now that your `index.html` is in your bucket, let's modify the program to have the bucket serve the file as a static website. First you need to set the `website` property on your bucket.
+
+```javascript
+const bucket = new aws.s3.Bucket("my-bucket", {
+    website: {
+        indexDocument: "index.html",
+    },
+});
+```
+
+Next, your `index.html` object will need two changes: an ACL of public-read so that it can be accessed anonymously over the Internet, and a content type so that it is served as HTML:
+
+```javascript
+const bucketObject = new aws.s3.BucketObject("index.html", {
+    acl: "public-read",
+    contentType: "text/html",
+    bucket: bucket,
+    source: new pulumi.asset.FileAsset(filePath),
+});
+```
+
+Finally, at the bottom of your program export the resulting bucket’s endpoint URL so you can easily access it:
+
+```javascript
+exports.bucketEndpoint = pulumi.interpolate`http://${bucket.websiteEndpoint}`;
+```
+
+{{% /choosable %}}
+
+{{% choosable language typescript %}}
+Now that your `index.html` is in your bucket, let's modify the program to have the bucket serve the file as a static website. First you need to set the `website` property on your bucket.
+
+```typescript
+const bucket = new aws.s3.Bucket("my-bucket", {
+    website: {
+        indexDocument: "index.html",
+    },
+});
+```
+
+Next, your `index.html` object will need two changes: an ACL of public-read so that it can be accessed anonymously over the Internet, and a content type so that it is served as HTML:
```

Comment:
```suggestion
Next, your `index.html` object will need two changes: an ACL of `public-read` so that it can be accessed anonymously over the Internet, and a content type so that it is served as HTML:
```